content/events/2020-online-get-together/rse-supporting-equality.md

```diff
@@ -16,3 +16,47 @@ have noticed after years of supporting researchers.  I will relate
 this to the services which can be provided by RSEs, and present a
 vision for addressing this by developing our own skills and promoting
 RSE services to our institutions.
+
+---
+
+#### Questions and comments
+
+- Perhaps getting sympathy from more traditional professionals in academia might not be easy because the RSE career path is not nearly (yet?) as "hard-coded" as that for, say, a professor.
+  - I agree with this. As an RSE, will I just need to leave academia sooner or later as all other "non professors" or do I just become the old IT staff member? *insert steve buscemi meme "hello fellow kids"* ![](https://i.imgur.com/VbDhUXZ.png)
+  - [name=a] A lot of people do leave, like me, but I don't think that's necessarily a problem. There needs to be people industry to work with RSEs and a goal I'd like to see is a good industry/RSE collaboration with RSEs being the people speaking the common language in partnerships. Quick plug for my talk on Wednesday containing some information on both sides of industry partnerships!
+
+- Apprenticeship can also happen by reading how others work (read their code, watch their code review, watch the tools they use)
+  - [name=speaker] yes!  I've learned plenty by reading the right blogs, for example.
+
+- Suggestion: using equality of opportunity for competent knowledge/skill rather than simply saying equality. e.g. we want to achieve equality of opportunity for everyone to be able to acquire knowledge rather than equalizing everyone's knowledge in a certain field.
+  - [name=speaker] Yes, that's correct.  We can't make the same outcome, but hopefully people have the same opportunity, without implicit prerequisites that some people don't have.
+
+- What is your take on implications of cultural differences on the topic of "supporting equality of opportunity". for example, many of your ideas seem to be easier to apply in a collectivist society as opposed to an individualist society which probably most of the Nordic could be categorized as.
+
```

Comment:
```suggestion
  - [name=speaker] This is not my speciality, but I think most of the points I make come about because we are very individualist and assume that everyone can make their own way.  That breaks down when not everyone has the network to do that.  Perhaps you could even say, those who think they succeed as individualists often happen to have these implicit networks that make that possible, yet doesn't get recognized.

```